bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/charge-channel-types.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mercedesme"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="charge-flap-channel">
+		<item-type>Number</item-type>
+		<label>Charge Flap</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Open</option>
+				<option value="1">Closed</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-ac-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler AC</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Plugged</option>
+				<option value="2">Unlugged</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-dc-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler DC</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Plugged</option>
+				<option value="2">Unlugged</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-lock-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler Lock</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Locked</option>
+				<option value="1">Unlocked</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="active-channel">
+		<item-type>Switch</item-type>
+		<label>Charge Active</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="power-channel">
+		<item-type>Number:Power</item-type>
+		<label>Charge Power</label>
+		<state pattern="%.1f kW" readOnly="true"/>
+	</channel-type>
+	<channel-type id="program-channel">
+		<item-type>Number</item-type>
+		<label>Charge Program</label>
+	</channel-type>
+	<channel-type id="end-time-channel">
+		<item-type>DateTime</item-type>
+		<label>Charge End Estimation</label>
+		<state pattern="%1$tH:%1$tM" readOnly="true"/>
+	</channel-type>
+	<channel-type id="max-soc-channel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Charge SoC Maximum</label>
+		<state pattern="%d %%"/>
+		<command>
+			<options>
+				<option value="50 %">50 %</option>
```

Comment:
My real system is still running on 4.0.3 while my test system is on 4.1.1 - both working. Any other suggestions how to define percentage options?

Reply:
Oh it can work perfectly fine. This is just an area that is historically very error prone. I would also consider it a bug in the UI if this is not working.

Reply:
Ok, working fine on my site and also didn't get complains in the community thread.
I'll leave it as it is.